integrity-backend/input-dropbox-examples/generic-meta-content.json

```diff
@@ -0,0 +1,24 @@
+{
+  "contentMetadata":
+  {
+    "name": "Inbound content from dropbox",
+    "mime": "application/zip",
```

Comment:
They are uploading a zip or a PDF/JPG?

If it's a ZIP I'd expect them to send along this file?